src/components/Button/Button.tsx

```diff
@@ -10,22 +10,22 @@ export interface IButtonState {
   ariaDescriptionId?: string;
 }
 
-let _instance = 0;
-
 export class Button extends React.Component<IButtonProps, IButtonState> {
   public static defaultProps: IButtonProps = {
     elementType: ElementType.button,
     buttonType: ButtonType.normal
   };
+  // Start at random value in case consumer bundles react multiple times
+  private static _instance: number = Math.round(Math.random() * 1000);
```

Comment:
Why don't we just create a simple helper called globalUtilities.ts or something, which has a getId method that derives it off a global namespaced value. That way if you did have 2 button instances on the page, they would share the state. Random numbers are error prone.